packages/core/src/components/TextField/Styled/TextField.styled.tsx

```diff
@@ -22,22 +33,34 @@ const styleWithLabel = ({ variant }: StyledProps) => {
     `;
 };
 
+const setFontStyle = ({ theme, size, multiline }: StyledProps) => {
+    if (multiline) {
+        return getFontStyle({ theme, fontVariant: 'body2' });
+    }
+    return getFontStyle({ theme, fontVariant: theme.textField.textVariant[size] });
+};
+
+const setHeight = ({ multiline }: StyledProps) => multiline && `height: 100%`;
+
 export const Input = styled('input')<StyledProps>`
     color: ${({ variant, theme }) => theme.textField[variant].default.textColor};
     width: 100%;
+    ${setHeight};
     margin: 0;
-    padding: 0;
+    padding: ${({ multiline, variant, size }) => (multiline && variant !== 'fusion' && size === 'M' ? '1.6rem 0 0 0' : 0)};
     box-sizing: border-box;
     transition: all 100ms ease-out;
     background: transparent;
     border: none;
     text-overflow: ellipsis;
+    resize: none;
     z-index: 1;
+    scroll-padding-top: 10px;
 
     ${({ isLabelPresent, size }) => isLabelPresent && size !== 'S' && styleWithLabel};
     &,
     & ~ ${MaskPlaceholder} {
-        ${({ theme, size }) => getFontStyle({ theme, fontVariant: theme.textField.textVariant[size] })}
+        ${setFontStyle}
```

Comment:
you can write directly over here 
`${({theme, size, multiline }) => getFontStyle({ theme, fontVariant: multiline ? 'body2' : theme.textField.textVariant[size] })}`